app/static/ie-status.json

```diff
@@ -1885,6 +1885,24 @@
     "id": 6049512976023552
   },
   {
+    "name": "WebGL 2.0",
+    "category": "Graphics",
+    "link": "https://www.khronos.org/registry/webgl/specs/latest/2.0/",
+    "summary": "An update to WebGL, based on OpenGL ES 3.0",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text": "Under Consideration ",
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "medium"
+    },
+    "msdn": "",
+    "wpd": "",
+    "demo": "https://www.youtube.com/watch?v=2v6iLpY7j5M",
```

Comment:
Not sure we want to link the youtube video. This fields is to link to demos hosted in http://dev.modern.ie/testdrive/ (although we haven't used it yet).
@jacobrossi what do you think?
In any case, cool stuff!